### PR TITLE
Update sample templates

### PR DESCRIPTION
**Pull request by SteveRosam**

### CODE REVIEW

File: python/empty/README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
-# Empty template
+# Sample template
 
-[This project]() is an empty template for Python applications using the Quix serverless environment.
+[This project](https://github.com/quixio/quix-samples/tree/main/python/empty) is a template for Python based Quix Samples using the Quix serverless environment.
```
Comment (tbedford) [inline python/empty/README.md:3]: > is a template for Python based Quix Samples

Should be Python-based. Perhaps better as:

...is a Python template for the Quix serverless environment.

Note: probably other instances too.

**Review by tbedford — CHANGES_REQUESTED** [review]
Looks good. 

I just wanted to be clear as I thought we decided on Quix Code Samples, and perhaps Code Samples?

Also, where we say "a Python-based template", we can just simplify to "a Python template", and then not worry about hyphenation.

**Comment by SteveRosam** [comment]
> I just wanted to be clear as I thought we decided on Quix Code Samples, and perhaps Code Samples?

Not sure what you're referring to here. Is there a specific problem somewhere?

**Comment by SteveRosam** [comment]
Hopefully all done now @tbedford .
This felt like scope creep on the ticket. Lets create separate tickets for things like this in future.

**Review by tbedford — APPROVED** [review]
LGTM!

**Pull request merged**